README.md

```diff
@@ -192,6 +195,12 @@ Default: `true`
 
 By setting `wait` to `true`, the build will wait until the triggered pipeline builds are successful before proceeding
 
+### `download` (optional)
+
+Default: `true`
+
+By setting `download` to `false`, the plugin assumes that the binary is present in `/usr/bin`.
```

Comment:
It would be useful to add use case on when you might one to use this config? Similar to the one you have mentioned on the PR comment.